app/src/main/kotlin/com/mapbox/vision/teaser/replayer/ReplayModeFragment.kt

```diff
@@ -0,0 +1,188 @@
+package com.mapbox.vision.teaser.replayer
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.*
```

Comment:
Do we need both imports, isn't `android.view.View.*` enough?

Reply:
ktlint catches it

Reply:
fixed